src/sentry/models/latestappconnectbuildscheck.py

```diff
@@ -0,0 +1,50 @@
+"""Database models to keep track of the App Store Connect builds for a project.
+
+For projects using the App Store Connect symbol source this keeps track of the last time we
+did manage to check for available builds.
+"""
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import FlexibleForeignKey, Model
+
+
+class LatestAppConnectBuildsCheck(Model):
+    """
+    The last date and time Sentry checked App Store Connect for new builds associated with a
+    specific appconnect source in a project.
+    """
+
+    __include_in_export__ = False
+
+    project = FlexibleForeignKey("sentry.Project", db_constraint=False)
+
+    # The symbol source's unique identifier in the project. Also known as the id on an
+    # AppStoreConnectConfig.
+    source_id = models.CharField(max_length=200)
+
+    # When sentry last checked App Store Connect for new builds. A check may return zero new builds,
+    # or no builds at all if all existing builds have expired.
+    last_checked = models.DateTimeField(default=timezone.now)
+
+    @classmethod
+    def update_date(cls, project, source_id, new_date):
+        if new_date is None:
+            new_date = timezone.now()
+        try:
+            latest_check = cls.objects.get(project=project, source_id=source_id)
+            latest_check.last_checked = new_date
+            latest_check.save(update_fields=["last_checked"])
```

Comment:
jfyi, we have a `.update`  you can use to do this - `latest_check.update(last_checked=new_date)`.